### DISCUSSION TITLE
change border color based on active display

### DISCUSSION THREAD

**Discussion by aspauldingcode** [FelixKratz/SketchyBar#603] [Q&A]
Hi!

I'm wondering how I can make the active display sketchybar's border blue, and inactive displays with sketchybar's border set to black. Is this already possible?

**Comment by FelixKratz** [comment]
You can do this by using a bracket that spans from the left most part of the bar to the right most part of the bar and set the items of the bracket to only show on the active display like this:
```bash
sketchybar --bar color=0xff000000 border_color=0xffffffff border_width=2 height=30 padding_left=0 padding_right=0

sketchybar --add item left_anchor left --set left_anchor width=0 display=active \
           --add item right_anchor right --set right_anchor width=0 display=active \
           --add bracket active_backdrop left_anchor right_anchor \
           --set active_backdrop background.color=0xffffffff background.border_color=0xff000000 background.border_width=2 background.height=30
```
This creates a black bar with white border on the inactive screens and a white bar with black border on the active screen. The anchor items must be added before any other item is added to the bar for it to span the entire width of the bar. Also see https://github.com/FelixKratz/SketchyBar/discussions/439 for related ideas using brackets.
